readme.md

```diff
@@ -728,6 +728,7 @@
 - [ApostropheCMS](https://github.com/apostrophecms/apostrophe) - Content management system with an emphasis on intuitive front end content editing and administration built on Express and MongoDB.
 - [Strapi](https://github.com/strapi/strapi) - Content Management Framework (headless-CMS) to build powerful APIs.
 - [Tipe](https://github.com/tipeio/tipe) - Developer-first content management system with GraphQL and REST API from a schema file.
+- [Factor](https://github.com/fiction-com/factor) - VueJS dashboard framework and Headless CMS
```

Comment:
I still think you could have put more effort. Vue.js is not correctly written. `Headless` => `headless`. And there should be a dot at the end.